shared/oval/mount_option_var_tmp_bind.xml

```diff
@@ -42,4 +46,22 @@
     <ind:instance datatype="int" operation="greater than or equal">
     1</ind:instance>
   </ind:textfilecontent54_object>
+
+  <!-- On newer system /etc/mtab has changed behavior and there is no info
+       about binded mounts anymore. An alternative is to check if
+       /proc/self/mountinfo has a mount in which the root of mount is /tmp and
+       mount point is /var/tmp -->
+
+  <ind:textfilecontent54_test check="all" check_existence="at_least_one_exists"
+  comment="Ensure /var/tmp is mounted to /tmp"
+  id="test_mount_option_var_tmp_bind_mountinfo" version="1">
+    <ind:object object_ref="object_mount_option_var_tmp_bind_mountinfo" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object comment="look for a mount with a /tmp root and a mount point in /var/tmp in /proc/self/mountinfo"
+  id="object_mount_option_var_tmp_bind_mountinfo" version="1">
+    <ind:filepath>/proc/self/mountinfo</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*(\S+\s){3}/tmp[\s]+/var/tmp[\S\s]*$</ind:pattern>
+    <ind:instance datatype="int" operation="greater than or equal">
+    1</ind:instance>
```

Comment:
@yuumasato nitpick here, but can you make this one-liner? It's readability is weird.